pkg/odo/cli/storage/storage.go

```diff
@@ -155,22 +156,23 @@ var storageDeleteCmd = &cobra.Command{
 		} else {
 			if componentName != "" {
 				mPath := storage.GetMountPath(client, storageName, componentName, applicationName)
-				fmt.Printf("Are you sure you want to delete the storage %v mounted to %v in %v component? [y/N] ", storageName, mPath, componentName)
+				util.LogAskf("Are you sure you want to delete the storage %v mounted to %v in %v component? [y/N]: ", storageName, mPath, componentName)
 			} else {
-				fmt.Printf("Are you sure you want to delete the storage %v that is not currently mounted to any component? [y/N] ", storageName)
+				util.LogAskf("Are you sure you want to delete the storage %v that is not currently mounted to any component? [y/N]: ", storageName)
 			}
 			fmt.Scanln(&confirmDeletion)
 		}
 		if strings.ToLower(confirmDeletion) == "y" {
 			componentName, err = storage.Delete(client, storageName, applicationName)
 			odoutil.CheckError(err, "failed to delete storage")
 			if componentName != "" {
-				fmt.Printf("Deleted storage %v from %v\n", storageName, componentName)
+				util.LogInfof("Deleted storage %v from %v", storageName, componentName)
 			} else {
-				fmt.Printf("Deleted storage %v\n", storageName)
+				util.LogInfof("Deleted storage %v", storageName)
 			}
 		} else {
-			fmt.Printf("Aborting deletion of storage: %v\n", storageName)
+			util.LogErrorf("Aborting deletion of storage: %v", storageName)
+			os.Exit(1)
```

Comment:
guess this can be removed too

Reply:
Same as https://github.com/redhat-developer/odo/pull/983/files/88ec602332333ff561899ad06bb2a1a5109a7e5f#r236993051